<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateSenatorDialog</class>
 <widget class="QDialog" name="CreateSenatorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>543</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <widget class="QWidget" name="containerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>91</y>
      <width>500</width>
      <height>360</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <widget class="QToolButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>473</x>
       <y>9</y>
       <width>18</width>
       <height>18</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>39</y>
       <width>500</width>
       <height>321</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QToolButton" name="okBtn">
       <property name="geometry">
        <rect>
         <x>137</x>
         <y>260</y>
         <width>96</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
      <widget class="QToolButton" name="cancelBtn">
       <property name="geometry">
        <rect>
         <x>267</x>
         <y>260</y>
         <width>96</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>150</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{font:  11px &quot;微软雅黑&quot;;color: rgb(83,107,215);}

</string>
       </property>
       <property name="text">
        <string>可用账户</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="accountComboBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>151</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QLabel" name="addressLabel">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>300</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <underline>false</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(192, 196, 212);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByMouse</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>90</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{font:  11px &quot;微软雅黑&quot;;color: rgb(83,107,215);}

</string>
       </property>
       <property name="text">
        <string>账户地址</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QStackedWidget" name="stackedWidget_fee">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>60</y>
         <width>221</width>
         <height>151</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="registerLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>150</width>
         <height>60</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{font:  11px &quot;微软雅黑&quot;;color: rgb(83,107,215);}

</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11px;&quot;&gt;已注册的非Candidate账户才可以成为Wallfacer，点击&lt;/span&gt;&lt;a href=&quot;showRegisterDialog&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;注册&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-size:11px;&quot;&gt;新账户&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
